server/adapters/mysql.go

```diff
@@ -255,11 +256,39 @@ func (m *MySQL) toDeleteQuery(conditions *DeleteConditions) (string, []interface
 	return strings.Join(queryConditions, conditions.JoinCondition), values
 }
 
+//Clean deletes all records in tableName table
+func (m *MySQL) Clean(tableName string) error {
+	wrappedTx, err := m.OpenTx()
+	if err != nil {
+		return err
+	}
+
+	if err := m.cleanTableInTransaction(wrappedTx, tableName); err != nil {
+		wrappedTx.Rollback()
+		return err
+	}
+
+	return wrappedTx.DirectCommit()
+}
+
 //Close underlying sql.DB
 func (m *MySQL) Close() error {
 	return m.dataSource.Close()
 }
 
+func (m *MySQL) cleanTableInTransaction(wrappedTx *Transaction, tableName string) error {
+	statement := fmt.Sprintf(mySQLCleanTableTemplate, m.config.Db, tableName)
+	m.queryLogger.LogDDL(statement)
+
+	_, err := wrappedTx.tx.ExecContext(m.ctx, statement)
```

Comment:
here the same as in BigQuery. We should check `error doesn't exist` error and omit it.